### DISCUSSION TITLE
blog/enhance-your-readme-with-asciinema

### DISCUSSION THREAD

**Discussion by giscus** [cesarsotovalero/cesarsotovalero.github.io#27] [Announcements]
# blog/enhance-your-readme-with-asciinema

In this tutorial, I show you to create beautiful terminal recordings that your users will love using ascinema.

https://www.cesarsotovalero.net/blog/enhance-your-readme-with-asciinema.html

**Comment by Albert-Moreno-derbysoft** [comment]
Great post, it's cool to give a bit more of color to README files! On the other hand, could you share your prompt! It's really cool!

**Reply by cesarsotovalero** [reply]
Hi @albert-ds, 
Thanks for your feedback! I just wrote a blog post for you that explains how I set up my terminal https://www.cesarsotovalero.net/blog/my-ultimate-terminal-customizations-for-macos.html Enjoy it 😄
